huaweicloud/resource_huaweicloud_elb_loadbalancer.go

```diff
@@ -0,0 +1,478 @@
+package huaweicloud
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+
+	"github.com/huaweicloud/golangsdk"
+	"github.com/huaweicloud/golangsdk/openstack/common/tags"
+	"github.com/huaweicloud/golangsdk/openstack/elb/v3/loadbalancers"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func ResourceLoadBalancerV3() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLoadBalancerV3Create,
+		Read:   resourceLoadBalancerV3Read,
+		Update: resourceLoadBalancerV3Update,
+		Delete: resourceLoadBalancerV3Delete,
```

Comment:
can the loadbalancer be imported?

Reply:
some attributes can not be fetched from GET, so will not provide import currently.